packages/algob/src/builtin-tasks/run.ts

```diff
@@ -56,6 +57,7 @@ function partitionIntoSorted(unsorted: string[]): string[][] {
 export async function runMultipleScripts(
```

Comment:
I see what we are no longer running multiple scripts at a time. We should rename the function to like `runScripts`

Reply:
We need to think about it before we remove it. I have created a task for this.:https://www.pivotaltracker.com/story/show/183849598